ecr-login/api/client.go

```diff
@@ -76,7 +76,11 @@ func (self *defaultClient) GetCredentials(serverURL string) (*Auth, error) {
 	if err != nil {
 		return nil, err
 	}
-	log.Debugf("Retrieving credentials for %s in %s (%s)", registry.ID, registry.Region, serverURL)
+	logrus.
+		WithField("registry", registry.ID).
```

Comment:
Should line 79 and 80 be one line? This looks weird to my eyes

Reply:
I much prefer it broken across multiple lines like this, but why not use [WithFields](https://godoc.org/github.com/sirupsen/logrus#Logger.WithFields)?

Reply:
```go
logrus.
	WithFields(Fields{
		"registry": registry.ID,
		"region": registry.Region,
```

This looks like it adds an extra line and extra indentation.

Reply:
I prefer the `WithFields` version, but don't have an issue with `WithField`.

Reply:
I'd remove the line break after the module name and write it as
```
logrus.WithFields(Fields{
 ...
```
But I won't object if you want to stick with the chained `WithField()` form.